src/Adapter/MSTest.TestAdapter/ObjectModel/UnitTestElement.cs

```diff
@@ -33,11 +33,6 @@ public UnitTestElement(TestMethod testMethod)
     /// </summary>
     public TestMethod TestMethod { get; private set; }
 
-    /// <summary>
-    /// Gets or sets a value indicating whether the unit test should be ignored at run-time.
-    /// </summary>
-    public bool Ignored { get; set; }
```

Comment:
This was never read (except in a unit test) and is adding noise to the code. The class is internal so I removed it altogether